src/NUnitTestAdapter/NUnit3TestAdapter.csproj

```diff
@@ -59,10 +59,6 @@
       <HintPath>..\..\packages\Microsoft.VisualStudio.TestPlatform.ObjectModel.0.0.3\lib\net35\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Mono.Cecil, Version=0.9.6.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756, processorArchitecture=MSIL">
```

Comment:
It's a horrible hack, without which we can't build!

* We need mono.cecil in the same directory as the engine. We make VS put it there by way of a reference.
* BUT that reference introduces the famous error about a missing member on ExtensionAttribute, because mono.cecil defines it's own attribute. VS __sometimes__ picks the wrong one and tries to use it.
* SO after I install nuget package for the engine, I remove the reference to mono.cecil from the test adapter project. It's still copied into the directory due to the reference in the install project, so everything works. Well, I said it was horrible! :-)
* As a side issue, I also remove the references to the two agent exes, but that's because we don't need them (and the x86 exe causes a warning)

We clearly need a long term solution to this problem if other folks are going to use the engine packages but for the adapter, this is a way to get it a release out at least. The users of the adapter only get a package with what they need in tools - no references required.